src/main/java/com/google/sps/servlets/SentimentServlet.java

```diff
@@ -57,7 +57,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String captions = captionService.getCaptionFromId(videoId);
 
     if (comments.isEmpty() && captions.isEmpty()) {
-      VideoAnalysis videoAnalysis = new VideoAnalysis.Builder()
+      VideoAnalysis videoAnalysis = VideoAnalysis.builder()
```

Comment:
There's one more call to Builder that'd need to get updated, it's at the end of the doGet method

Reply:
Done, thanks. Not sure why I didn't get a build error from mvn for this.